.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Docker Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release_config_builder:
+    name: Release Cass Config Builder
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Login to DockerHub
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+      - name: Setup Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v3
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Publish to Dockerhub
+        run: |
+          GITHUB_REPO_URL="https://github.com/${{ github.repository }}"
+
+          VERSION_NUMBER="$(cat version.txt | tr -d '[:space:]')"
+          RELEASE_VERSION="${VERSION_NUMBER}"
+
+          DOCKERHUB_REPOSITORY="datastax/cass-config-builder"
+
+          # Make sure the version number of the project aligns with
+          # the tag that we have to prevent confusion.
+          GIT_TAG="${GITHUB_REF##*/}"
+          if ! [ "v${VERSION_NUMBER}" = "${GIT_TAG}" ]; then
+            echo "Git tag $GIT_TAG does not align with version number ${VERSION_NUMBER} in version.txt"
+            exit 1
+          fi
+
+          DOCKERHUB_TAGS=(--tag "${DOCKERHUB_REPOSITORY}:${RELEASE_VERSION}")
+          DOCKERHUB_UBI_TAGS=(--tag "${DOCKERHUB_REPOSITORY}:${RELEASE_VERSION}-ubi7")
```

Comment:
Ideally, we'd not rebuild everything here, but GH packages do not support multiarch images, so there isn't a good way to push a multiarch image to dockerhub without rebuilding using buildx.